tests/unit/test-cmd/test.build.js

```diff
@@ -329,4 +369,41 @@ describe('build', () => {
 
   });
 
+  describe('defaultPackageCreator', () => {
+
+    it('should reject on Unexpected errors', () => withTempDir(
```

Comment:
we should keep the coding style of this test consistent with the other tests in the same file:
```js
it('should reject on Unexpected errors', () => {
  return withTempDir(
     (tmpDir) => {
       ...
     });
});
```